intervention_compassion/models/compassion_intervention.py

```diff
@@ -59,6 +58,9 @@ class CompassionIntervention(models.Model):
         'compassion.project', 'icp_interventions', 'intervention_id', 'icp_id',
         string='ICPs', readonly=True
     )
+    product_id = fields.Many2one('product.template',
```

Comment:
I would name this field `product_template_id` since it is not a product but a product template.